grr/server/grr_response_server/gui/ui/components/timestamp/relative_timestamp_pipe.ts

```diff
@@ -0,0 +1,54 @@
+import {Pipe, PipeTransform} from '@angular/core';
+import {DateTime} from 'luxon';
+
+@Pipe({
+  name: 'relativeTimestamp'
+})
+export class RelativeTimestampPipe implements PipeTransform {
+  readonly oneDay = 86400000; // One day in millis
+  readonly oneHour = 3600000; // One hour in millis
+  readonly oneMin = 60000; // One minute in millis
+  readonly oneSecond = 1000; // One second in millis
+
+  transform(jsDate: Date, absoluteOnly: boolean): any {
+    if (jsDate === undefined) {
+      return "-";
+    }
+
+    const date = DateTime.fromJSDate(jsDate);
+
+    if (absoluteOnly) {
+      return this.getAbsoluteTimestamp(date);
+    }
+
+    const now = DateTime.local();
+    const timeElapsed = now.diff(date, "milliseconds").as("milliseconds");
+
+    if (timeElapsed < this.oneMin / 2) {
+      return "Just now";
+    }
+
+    if (timeElapsed < this.oneMin) {
+      return Math.floor(timeElapsed / this.oneSecond) + " seconds ago";
```

Comment:
Use template strings instead of string concatenation:
```
return `${Math.floor(timeElapsed / this.oneSecond)} seconds ago`;
```

Reply:
Didn't know about that, thanks!